src/App.tsx

```diff
@@ -15,71 +16,81 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type State = {
+  selectedItem: string | null;
+};
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export class App extends Component {
+  state: Readonly<State> = {
+    selectedItem: 'Jam',
+  };
 
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  switcher(item: string) {
+    if (item === this.state.selectedItem) {
+      this.setState({ selectedItem: null });
+    } else {
+      this.setState({ selectedItem: item });
+    }
+  }
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+  render() {
+    const { selectedItem } = this.state;
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+    return (
+      <main className="section container">
+        <h1
+          className={classNames(
+            'title',
+            { 'is-flex is-align-items-center': selectedItem },
+          )}
+        >
+          {selectedItem ? `${selectedItem} is selected` : 'No goods selected'}
 
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
+          {selectedItem && (
+            <>
+              {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
```

Comment:
let's try removing it

Reply:
Removing what?

Reply:
https://github.com/mate-academy/react_goods-selector/blob/master/src/App.tsx#L25